android_beta/8.0beta.yml

```diff
@@ -1,12 +1,13 @@
 release:
   release_date: '2024-09-30'
   text: |
-    **These notes apply to Thunderbird for Android version 8.0 beta 3 released October, 14, 2024.**
+    **These notes apply to Thunderbird for Android version 8.0 betas**
```

Comment:
This is probably fine but we use this line to mention the release date of the latest beta.

Reply:
Ah good to know, I was wondering if it was jut out of date or part of testing. Can update this in the next beta